<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please Input Password</string>
  </property>
  <widget class="QPushButton" name="m_btnOK">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>200</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnEditPwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>166</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnCancel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>200</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>13</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>New Password:</string>
      </property>
      <property name="buddy">
       <cstring>m_edtNewPwd</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="m_edtConfirmPwd">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Confirm Password:</string>
      </property>
      <property name="buddy">
       <cstring>m_edtConfirmPwd</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="m_edtCurrPwd">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Password:</string>
      </property>
      <property name="buddy">
       <cstring>m_edtCurrPwd</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="m_edtNewPwd">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_edtCurrPwd</tabstop>
  <tabstop>m_edtNewPwd</tabstop>
  <tabstop>m_edtConfirmPwd</tabstop>
  <tabstop>m_btnEditPwd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
